### PR TITLE
572 update chat styles avatars

### PR DESCRIPTION
**Pull request by pmarsh-scottlogic**
## Description

adds avatars in line with our UI update

### Regular messages
image

### Message is blocked by a defence
image

### Error in backend
image

### User message transformed
image

### level info
image

## Notes

- removes `SYSTEM` and `FUNCTION_CALL` from the `CHAT_MESSAGE_TYPE` enum. Pretty certain they were unused in the frontend.

## Concerns

- Are tests necessary? Hard to tell. If you think so, then what would those tests look like?

## Checklist

Have you done the following?

- [x] Linked the relevant Issue
- [ ] Added tests
- [x] Ensured the workflow steps are passing

### CODE REVIEW

File: frontend/src/components/ChatBox/ChatBoxMessage/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	align-items: center;
+}
+
+.avatar-circle-user {
+	background-color: #dff4f5;
+}
+
+.avatar-circle-bot {
+	background-color: #eeeeee;
+}
```
Comment (gsproston-scottlogic) [inline frontend/src/components/ChatBox/ChatBoxMessage/Avatar.css:17]: Colours should be in the theme file.

File: frontend/src/components/ChatBox/ChatBoxMessage/ChatBoxMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			? 'bot'
+			: message.type === CHAT_MESSAGE_TYPE.BOT_BLOCKED ||
+			  message.type === CHAT_MESSAGE_TYPE.ERROR_MSG
+			? 'botError'
+			: 'none';
+
+	const owner = avatar === 'botError' ? 'bot' : avatar;
+
+	const position =
+		owner === 'user' ? 'right' : owner === 'bot' ? 'left' : 'centre';
```
Comment (gsproston-scottlogic) [inline frontend/src/components/ChatBox/ChatBoxMessage/ChatBoxMessage.tsx:23]: Very minor, but I personally found this a little tricky to read. Might as well not use `owner` imo and just put it right into the `position` conditional.
```suggestion
	const position =
			avatar === 'user' 
			? 'right' 
			: avatar === 'bot' || avatar === 'botError'
			? 'left' 
			: 'centre';
```

**Review by gsproston-scottlogic — CHANGES_REQUESTED** [review]
Looks spot on! Just a request on theme colours from me. DW about the other comment if you prefer how it currently is.

File: frontend/src/components/ChatBox/ChatBoxMessage/ChatBoxMessage.tsx (hunk 2 above)
Comment (pmarsh-scottlogic) [reply frontend/src/components/ChatBox/ChatBoxMessage/ChatBoxMessage.tsx:23]: Ah, a relic from a previous implementation :) good spot

**Review by dhinrichs-scottlogic — APPROVED** [review]
Looks good! The avatars really add something, especially the Error bot image :D 
Good work :)

**Pull request merged**